demos/python_demos/human_pose_estimation_3d_demo/human_pose_estimation_3d_demo.py

```diff
@@ -0,0 +1,147 @@
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from argparse import ArgumentParser, SUPPRESS
+import json
+import os
+
+import cv2
+import numpy as np
+
+from modules.inference_engine import InferenceEngine
+from modules.input_reader import InputReader
+from modules.draw import Plotter3d, draw_poses
+from modules.parse_poses import parse_poses
+
+
+def rotate_poses(poses_3d, R, t):
+    R_inv = np.linalg.inv(R)
+    for pose_id in range(poses_3d.shape[0]):
+        pose_3d = poses_3d[pose_id].reshape((-1, 4)).transpose()
+        pose_3d[0:3] = np.dot(R_inv, pose_3d[0:3] - t)
+        poses_3d[pose_id] = pose_3d.transpose().reshape(-1)
+
+    return poses_3d
+
+
+if __name__ == '__main__':
+    parser = ArgumentParser(description='Lightweight 3d human pose estimation demo. '
+                                        'Press esc to exit, "p" to (un)pause video or process next image.',
+                            add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS,
+                      help='Show this help message and exit.')
+    args.add_argument('-m', '--model',
+                      help='Required. Path to an .xml file with a trained model.',
+                      type=str, required=True)
+    args.add_argument('-i', '--input',
+                      help='Required. Path to input image, images, video file or camera id.',
+                      nargs='+', default='')
+    args.add_argument('-d', '--device',
+                      help='Optional. Specify the target device to infer on: CPU, GPU, FPGA, HDDL or MYRIAD. '
+                           'The demo will look for a suitable plugin for device specified '
+                           '(by default, it is CPU).',
+                      type=str, default='CPU')
+    args.add_argument('--height_size', help='Optional. Network input layer height size.', type=int, default=256)
+    args.add_argument('--extrinsics_path',
+                      help='Optional. Path to file with camera extrinsics.',
+                      type=str, default=None)
+    args.add_argument('--fx', type=np.float32, default=-1, help='Optional. Camera focal length.')
+    args.add_argument('--no_show', help='Optional. Do not display output.', action='store_true')
+    args = parser.parse_args()
+
+    if args.input == '':
+        raise ValueError('Please, provide input data.')
+
+    stride = 8
+    net = InferenceEngine(args.model, args.device, stride)
+    canvas_3d = np.zeros((720, 1280, 3), dtype=np.uint8)
+    plotter = Plotter3d(canvas_3d.shape[:2])
+    canvas_3d_window_name = 'Canvas 3d'
+    cv2.namedWindow(canvas_3d_window_name)
+    cv2.setMouseCallback(canvas_3d_window_name, Plotter3d.mouse_callback)
+
+    file_path = args.extrinsics_path
+    if file_path is None:
+        file_path = os.path.join('data', 'extrinsics.json')
```

Comment:
This file should be located relative to the directory of the script itself.

Reply:
It is already there, `data` is on the same level with demo, isn't it?

Reply:
The issue is not with where the file is, but with how the demo locates it. You're trying to find it relative to the current directory, while you should be trying to find it relative to the script directory.